pytest_splunk_addon/splunk.py

```diff
@@ -209,6 +164,24 @@ def splunk(request):
     yield splunk_info
 
 
+@pytest.fixture(scope="session")
+def docker_compose_files(pytestconfig):
+    """
+    Get an absolute path to the  `docker-compose.yml` file. Override this
+    fixture in your tests if you need a custom location.
+
+    Returns:
+        string: the path of the `docker-compose.yml` file
+
+    """
+    docker_compose_path = os.path.join(
+        str(pytestconfig.invocation_dir), "docker-compose.yml"
+    )
+    LOGGER.info("docker-compose path: %s", docker_compose_path)
+
+    return [docker_compose_path]
```

Comment:
why is this returning a list? there is only one path that can be returned.

Reply:
The fixture is required for lovely-pytest-docker and it returns a list by default which will be then used by the module to spin up the containers. A similar example is provided at https://pypi.org/project/lovely-pytest-docker/